src/main/java/edu/texas/social/computing/housing/matching/DefaultMatching.java

```diff
@@ -0,0 +1,181 @@
+package edu.texas.social.computing.housing.matching;
+
+import edu.texas.social.computing.housing.objects.Agent;
+import edu.texas.social.computing.housing.objects.House;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.Stack;
+
+public class DefaultMatching implements MatchingStrategy {
+    List<Agent> agents;
```

Comment:
did we lose the methods Tiffany had written to generate the global states?

Reply:
Yes. I added back in later commits matching better what she was doing and what I am doing with objects instead of integers. I also added her old implementation back as a reference file.